### DISCUSSION TITLE
Could not resolve host: username

### DISCUSSION THREAD

**Discussion by StrangeMeta** [community/community#106679] [New to GitHub] [answered]
I've made a private repo, but I can't clone it. I'm using git clone https://StrangeMeta/accesstoken/github.com/repolink but it just says unable to access repolink : Could not resolve host: StrangeMeta.

**Comment by Olayiwolaaa** [comment]
make sure you are logged in on your cli. If not, then do this:
``` bash
     git init
     git config --global user.email "your-email@mailprovider.com
     git config --global user.name "your-username"
     git clone https://github.com/your-username/your-repository.git
```
## If this works, please make as answer.

**Reply by StrangeMeta** [reply]
I did that and it says repository not found. I'm using git bash, I don't know if that makes a difference?

**Comment by StrangeMeta** [comment, marked as answer]
Okay, I just kept on trying various orders of the segments of the command I put in the title and this one worked. Not sure why but, hey, it worked.
git clone https://accesstoken@github.com/username/reponame.git

**Reply by Olayiwolaaa** [reply]
You left the part where it was a token accessible private repository you created 😒. I needed that answer to get my badge man

**Reply by Phuchong0259** [reply]
ไม่เแาก็ได้ครับผมเรียนน้อยทำๆม่เป็นจริงๆๆ  โทรศัพท์แบตก็จะหมดแล้ว  ขอบคุณนะครับที่ไห้โอกาศแต่ผมอ่านไม่ออกจริง

**Reply by Olayiwolaaa** [reply]
โอ้ ฉันพูดว่า:

คุณออกจากส่วนที่เป็นพื้นที่เก็บข้อมูลส่วนตัวที่สามารถเข้าถึงได้ด้วยโทเค็นที่คุณสร้างขึ้น 😒 ฉันต้องการคำตอบนั้นเพื่อจะได้ตราสัญลักษณ์ของฉัน
ฮ่าๆ ฉันหวังว่าคำตอบของฉันจะเป็นประโยชน์ และฉันอาจถูกทำเครื่องหมายความคิดเห็นว่าเป็นคำตอบได้

**Comment by Otis148** [comment]
Could someone help me get back into my account. I opened it back a few years. I was given a 12 word password. went in and out a few times. Started to mine I believe and then just let it be on its own. And no matter what I try the last 1-1/2 years it will not acknowledge me and let me in. Of course this platform does not have a support team so I dont know what to do. Let me know by email or so if there is anything I can do. This automated system just keeps me going in circles. My email is otuttle@shaw.ca or OT.model57@gmail.com
i would appreciate it very much if I could access my account and move my crypto out. Thanks.

**Reply by gdanielgallego** [reply]
Leave
